conda_build/skeletons/cpan.py

```diff
@@ -5,12 +5,12 @@
 from __future__ import absolute_import, division, print_function
 
 import codecs
-from pkg_resources import parse_version
+import distutils.version
```

Comment:
I would recommend to import using ``from`` here
```suggestion
from distutils import version
```

Reply:
Or just import ``distutils``

Reply:
I was trying to do the minimal required import to wrap `LooseVersion`.  We aren't using all of `distutils`, only parts that are inside `distutils.version`.  

I think that `from distutils import version` will conflict with the many uses of `version` inside `def`s below, won't it?

If I end up changing `def LooseVersion` to `def loose_version` as you suggest below, I'll change this back to `from distutils.version import LooseVersion` as it was pre PR #2156